### DISCUSSION TITLE
I have my slingbox 500 userid/adminid from my account. Can i setup slinger server later?

### DISCUSSION THREAD

**Discussion by jaybeebradley** [GerryDazoo/Slinger#251] [Q&A]
I went to 
https://newwatchsecure.slingbox.com/watch/slingAccounts/account_boxes_js

and copied all of the info/text string.   Have not had a chance to setup my slinger server - will likely do on Mac and Win10,.

Just want to be sure I can setup my slinger server after tomorrow Nov. 9 when slingbox servers are supposed to be shutdown.

I.e. other than the contents of my account/box id's and passwords on the link to my account above - do i need to do anything else while slingbox servers still live?

Thanks so much for your thoughts/answers.  Sorry for the likely redundant question. :)

**Comment by AndyJStubbs** [comment]
If you can I would suggest at least setting up the remote control for whichever stb you use if you have not already done so prior to the shutdown.

**Reply by jaybeebradley** [reply]
THanks Andy!  Ok - that means basic server setup/config.ini editing to get that done?

And do we think sling servers go down at midnight tonight or sometime tomorrow - nov. 9, or tomorrow at midnight?  argh!

**Reply by AndyJStubbs** [reply]
Sorry what I meant is using the slingplayer app / configuration to make sure that you have configured your remote control prior to the big kill switch in the sky getting flicked. 
That way the box will know which remote codes to accept and should be fairly simple to set up using Gerry's slinger. 
If you already have the remote working using the slingplayer app then you should be on fairly safe ground.

If your slingbox is currently working correctly and accepting remote control commands then it shouldnt matter much at which point you configure slinger to work with it.

**Reply by jaybeebradley** [reply]
thanks.  so i use it on a laptop firefox tab connected to a tivo mini box.  when i use the web slingplayer and click the remote symbol a little tivo remote pops out and i use my trackpad to click on that remote and it works to control the tivo - via the slingbox.

so does that mean i'm set since it's worked for me in past and all that info is 'in the slingbox' already?   or is there something else?

**Reply by AndyJStubbs** [reply]
Yup you should be good - I think there are also several tivo custom remotes available already from various contributors so you should be fine
